### ISSUE TITLE
HUD suggestions

### ISSUE THREAD

**Issue by KryptonCaptain** [Mixac1/DangerRPG#4]
Changing the position and scale of the HUD elements is needed so players can customise their screen. There are many other mods that also add things to the HUD, and being able to move them around is very important so they do not overlap, or take up too much of the screen/do not block things or get in the way for the player
(As in, configs to change X and Y offset or position)

![drpg hud](https://cloud.githubusercontent.com/assets/8790829/18592329/ddabcc58-7c79-11e6-8525-367a0b95c166.png)

As a concept to change the player stats HUD, I put this together.
- Player portrait is removed to save space. If player wants to see themselves, just open inventory or change perspective
- Hunger is no longer a vertical bar in a weird place, and now sits under health bar. If possible, hunger bar would also displays food saturation as overlay similar to mod [AppleCore](https://minecraft.curseforge.com/projects/applecore)
- Armor bar for physical and magic resistance are merged, and put on top of each other to save space. Could maybe change colour of magic resistance bar to a brighter purple colour
![drpg hud 2](https://cloud.githubusercontent.com/assets/8790829/18592501/977e0db2-7c7a-11e6-8ac4-59d74b02a3f7.png)

**Comment by Mixac1** [comment]
Nice ideas.

I will make setting of HUD in config. 

But what about air bar?

**Comment by KryptonCaptain** [comment]
I totally forgot about the air bar. I don't play underwater much in minecraft
Had to check that you'd changed it, since normal position of it is fine for me.
I suppose it could just go horizontal, under armour bars

**Comment by KryptonCaptain** [comment]
New HUD is very nice. The option to toggle through modes/settings is good
I was a bit confused to start with, when I saw numbers for health and mana were gone, but found them once I changed HUD mode.

But I am unsure what these config options change, as I do not see any difference in the HUDs
-    B:guiPlayerHUDIsInvert=false
-    B:guiEnemyHUDIsInvert=true

**Comment by Mixac1** [comment]
What are you think about new HUD?

**Comment by Mixac1** [comment]
Okey, I will add it.

**Comment by Mixac1** [comment]
I have just added its setting in config.

**Comment by KryptonCaptain** [comment]
So, I tried testing new setting, but on loading world after changing config, it does something very weird to your HUD. Where it normally is, is all missing or broken. It resets if you cycle through modes with the key, but is still very strange.
I changed config to _I:guiDafaultHUDMode=2_ from =1
![2016-09-20_21 32 15](https://cloud.githubusercontent.com/assets/8790829/18668649/f1c79048-7f79-11e6-8615-3dde0e4b35f9.png)
